CHANGELOG.md

```diff
@@ -48,6 +48,7 @@ The format is loosely based on [Keep a Changelog](http://keepachangelog.com/en/1
 * Performance regression: Adding setups to a mock becomes slower with each setup (@CeesKaas, #1110)
 
 * Regression: `mock.Verify[All]` no longer marks invocations as verified if they were matched by conditional setups. (@Lyra2108, #1114)
+* Newly introduced: `AmbiguousMatchException` raised when interface has property indexer besides property in VB. (@mujdatdinc, #1129)    
```

Comment:
You've made your commit against an outdated version of the `main` branch. If you look at [the current `CHANGELOG.md`](https://raw.githubusercontent.com/moq/moq4/main/CHANGELOG.md) you'll note that the top `## Unreleased` section here has since become `## 4.16.0 (2021-01-16)`.

Please rebase your commit on top of the current `main` branch, and add your changelog entry at the top of the file, as follows:

```diff
 [...]
 The format is loosely based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+
+## Unrelease
+
+#### Fixed
+
+* `AmbiguousMatchException` raised when interface has property indexer besides property in VB. (@mujdatdinc, #1129)
+

 ## 4.16.0 (2021-01-16)
 [...]
```